.ci/openshift-ci/README.md

```diff
@@ -0,0 +1,116 @@
+should be created the Prow job configuration files.
+
+The CI operator has this [jobs directory](https://github.com/openshift/release/tree/master/ci-operator/jobs)
+on which Prow job files are placed on also per-component/repository/branch
+basis. Find the `kata-containers` jobs directory under [CI operator jobs](https://github.com/openshift/release/tree/master/ci-operator/jobs/).
+These files can be automatically generated from the CI operator configuration
+files as follows:
+
+```
+$ git clone https://github.com/openshifit/release
+$ cd release
+$ make jobs
+```
+
+# The CI pipeline explained
+
+The OpenShift CI pipeline implemented to test Kata Containers can be
+logically broken into the following stages:
+
+  * [Build container images](#build-container-images)
+  * [Create the test cluster](#create-the-test-cluster)
+  * [Install Kata Containers](#install-kata-containers)
+  * [Run tests](#run-tests)
+
+## Build container images
+
+In this initial stage container images are prepared to be used along the
+pipeline. For instance, the base OpenShift image version or the build root image
+can be chosen. The preparation instructions are declared in the
+Kata Containers CI operator configuration file.
+
+The following images are built (orderly):
+
+1. `pipeline:src` - wraps the source repository
+    * The source repository is copied to `$KATA_SRC/<repo>` in the image. Assume
+that `$KATA_SRC` is `/go/github.com/src/kata-containers`
+    * The repository and branch are inferred by the CI operator unless
+explicitly declared
+2. `pipeline:bin` - wraps the built-from-source artifacts
+    * Uses the source repository packaged in `pipeline:src`
+    * The `$KATA_SRC/<repo>/.ci/openshift-ci/build.sh` script is called to build
+and install Kata Containers in the image, and the destination directory is `$KATA_SRC/<repo>/_out/build_install`
+3. `pipeline:kata-installer` - wraps the installation scripts and payload
+    * This image is built from the `$KATA_SRC/<repo>/.ci/openshift-ci/images/`
+Dockerfile context directory
+    * The CI operator is instructed to map the Kata Containers installed
+directory from `pipeline:bin` to `_out` in that context directory. This
+constitutes the packaged Kata Containers artifacts which will be installed in
+the test cluster.
+
+## Create the test cluster
+
+By the means of [templates](https://github.com/openshift/release/tree/master/ci-operator/templates)
+the CI operator allows for preparing the test environment. Commonly those
+templates are made to instruct the operator on how to deploy and configure
+OpenShift in a given cloud provider, then run end-to-end tests.
+
+In special for Kata Containers the used template needs to enable nested
+virtualization in the test cluster. See the template currently used in the
+Kata Containers CI operator configuration file.
+
+## Install Kata Containers
+
+In this stage the test cluster should get Kata Containers installed in. In other
```

Comment:
s/In other to achieve/In order to achieve/ ?

Reply:
yet another good catch. thanks!

Reply:
done